test/OrchardCore.Abstractions.Tests/Modules/StringExtensionsTests.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using Xunit;
+
+namespace OrchardCore.Modules
+{
+    public class StringExtensionsTests
+    {
+        [Fact]
```

Comment:
We could use `Theory` for both test case 1 & 3

Reply:
Ok, using `Theory` in test case 1 & 3.